docs/content/integrations/duckdb/reference.mdx

```diff
@@ -0,0 +1,596 @@
+        {
+            "date": DailyPartitionsDefinition(start_date="2023-01-01"),
+            "species": StaticPartitionDefinition(
+                ["Iris-setosa", "Iris-virginica", "Iris-versicolor"]
+            ),
+        }
+    ),
+    metadata={"partition_expr": {"date": "TO_TIMESTAMP(TIME)", "species": "SPECIES"}},
+)
+def iris_dataset_partitioned(context) -> pd.DataFrame:
+    partition = partition = context.partition_key.keys_by_dimension
+    species = partition["species"]
+    date = partition["date"]
+
+    # get_iris_data_for_date fetches all of the iris data for a given date,
+    # the returned dataframe contains a column named 'time' with that stores
+    # the time of the row as an integer of seconds since epoch
+    full_df = get_iris_data_for_date(date)
+
+    return full_df[full_df["species"] == species]
+
+
+@asset
+def iris_cleaned(iris_dataset_partitioned: pd.DataFrame):
+    return iris_dataset_partitioned.dropna().drop_duplicates()
+```
+
+Dagster uses the `partition_expr` metadata to craft the `SELECT` statement when loading the correct partition in a downstream asset. For multi-partitions, Dagster concatenates the `WHERE` statements described in the above sections to craft the correct `SELECT` statement.
+
+A partition must be selected when materializing the above assets, as described in the [Materializing partitioned assets](/concepts/partitions-schedules-sensors/partitions#materializing-partitioned-assets) documentation. For example, when materializing the `2023-01-02|Iris-setosa` partition of the above assets, the following query will be used:
+
+```sql
+SELECT *
+ WHERE SPECIES in ('Iris-setosa')
+   AND TO_TIMESTAMP(TIME) >= '2023-01-02 00:00:00'
+   AND TO_TIMESTAMP(TIME) < '2023-01-03 00:00:00'
+```
+
+In this example, the data in the `TIME` column are integers, so the `partition_expr` metadata includes a SQL statement to convert integers to timestamps. A full list of DuckDB functions can be found [here](https://duckdb.org/docs/sql/functions/overview).
+
+</TabItem>
+</TabGroup>
+
+---
+
+## Storing tables in multiple schemas
+
+You may want to have different assets stored in different DuckDB schemas. The DuckDB I/O manager allows you to specify the schema in several ways.
+
+If you want all of your assets to be stored in the same schema, you can specify the schema as configuration to the I/O manager, as we did in [Step 1: Configure the DuckDB I/O manager](/integrations/duckdb/using-duckdb-with-dagster#step-1-configure-the-duckdb-io-manager) of the [Using Dagster with DuckDB tutorial](/integrations/duckdb/using-duckdb-with-dagster).
+
+If you want to store assets in different schemas, you can specify the schema as part of the asset's asset key. The schema will be the second-to-last value in the asset key. For `SourceAsset`s, the full asset key is specified using the `key` parameter. For Software-defined Assets, the full asset key is the `key_prefix` prepended to the asset name.
```

Comment:
```suggestion
If you want to store assets in different schemas, you can specify the schema as part of the asset's asset key:

- **For `SourceAsset`**, use the `key` parameter. The schema should be the second-to-last value in the parameter. In the following example, this would be `daffodil`.

- **For Software-defined Assets**, use the `key_prefix` parameter. This value will be prepended to the asset name to create the full asset key. In the following example, this would be `iris`.
```

Thoughts? I know it's a bit repetitive with the explanation following the example, but I think it might make things click while reading the code instead of immediately after. Definitely not tied to this tho

Reply:
i really like that!